etc/profile-m-z/torbrowser-launcher.profile

```diff
@@ -53,12 +53,14 @@ nou2f
 novideo
 protocol unix,inet,inet6
 seccomp !chroot
+seccomp.block-secondary
 #tracelog - may cause issues, see #1930
 
 disable-mnt
 private-bin bash,cat,cp,cut,dirname,env,expr,file,gpg,grep,gxmessage,id,kdialog,ln,mkdir,mv,python*,rm,sed,sh,tail,tar,tclsh,test,tor-browser,tor-browser-en,torbrowser-launcher,update-desktop-database,xmessage,xz,zenity
 private-dev
 private-etc @tls-ca
+private-opt tor-browser
```

Comment:
Can cause slow startup.

Breaks if any other name is used (e.g. torbrowser-launcher).

Reply:
Better revert it in that case. Thanks for the info!